doc/ranger.pod

```diff
@@ -328,6 +328,14 @@ the end of the line automatically.
 
 To write a literal %, you need to escape it by writing %%.
 
+For adding a space character after a chained command, you also need to escape
```

Comment:
I think it'd be more appropriate trailing the `%space` explanation. Maybe change the order so the %% has already been explained.

"Note that macros are expanded twice when using chain. For example, to use `%space` with a chain command you'd write `%%space`."

I think that's clearer. Also, I don't like your example because `%space` is just a workaround for editors trimming trailing whitespace. So you'd normally only need it at the very end of commands you put in `rc.conf`.

Reply:
I've implemented your comments in the next commit.